drop-in/src/main/res/values/strings.xml

```diff
@@ -12,15 +12,16 @@
     <string name="action_failed">There was an error while processing your payment. Please try again later.</string>
     <string name="time_out_error">Request timed out. Please try again.</string>
     <string name="unknown_error">An unknown error occurred</string>
-    <string name="store_payment_methods_header">Your payment methods</string>
-    <string name="other_payment_methods">Select other method</string>
+    <string name="store_payment_methods_header">Stored</string>
+    <string name="other_payment_methods">others</string>
```

Comment:
Should this be `Others` with capital first letter?

Reply:
It probably should, but it comes from translations like this.